sdk/src/camera_96tof1.cpp

```diff
@@ -61,9 +61,12 @@ aditof::Status Camera96Tof1::setMode(const std::string &mode,
 
     LOG(INFO) << "Chosen mode: " << mode.c_str();
 
-    std::vector<std::pair<std::string, int>> modeRanges = {
+    std::vector<std::pair<std::string, int>> modeRangesMax = {
```

Comment:
You could create a struct which can describe a range. Make it internal to this class. And have just one array. You won't need to use the second find_if either.

Reply:
done, thanks